scripts/sct_run_batch.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+##############################################################################
+#
+# Wrapper to processing scripts, which loops across subjects. Data should be
+# organized according to the BIDS structure:
+# https://github.com/sct-pipeline/spine_generic#file-structure
+#
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Authors: Julien Cohen-Adad, Chris Hammill
+#
+# About the license: see the file LICENSE.TXT
+##############################################################################
+
+from __future__ import absolute_import
+
+import os
+import sys
+import argparse
+import numpy as np
+import multiprocessing
+import subprocess
+import re
+import time
+
+import spinalcordtoolbox.image as msct_image
+from spinalcordtoolbox.image import Image, concat_data
+from spinalcordtoolbox.utils import Metavar, SmartFormatter
+
+parser = argparse.ArgumentParser(
+  description = 'Wrapper to processing scripts, which loops across subjects. Data should be '
+                'organized according to the BIDS structure: '
+                'https://github.com/sct-pipeline/spine_generic#file-structure.\n'
+                'The processing script (task) should accept a subject directory as its only argument. '
+                'Additional information is passed via environment variables \n'
+                'PATH_SEGMANUAL (--path-segmanual)\n'
+                'PATH_DATA (--data-path)\n'
```

Comment:
I'm wondering if we should not rename it "FOLDER_DATA" (same for LOG and QC), to make it explicit that here we expect a folder (not a full path to a folder).

Reply:
I prefer PATH_DATA, folder as far as I know is windows terminology for path. On the other hand there is definitely a UX argument that many users on mac/windows would be more familiar with the name folder instead of path. 

I'll leave the decision to you, keeping PATH for now. 

Additionally this flag can handle both relative and absolute paths so I'm not sure I understand your last point.

Reply:
@jcohenadad can you clarify? Thanks

Reply:
> Additionally this flag can handle both relative and absolute paths so I'm not sure I understand your last point.

you're right-- i was actually making the point for PATH_RESULTS, PATH_LOG, etc. which are actually single "folder" (not paths with several folder depths). But yes, PATH_DATA should stay PATH_DATA